.github/workflows/common-post-jobs.yaml

```diff
@@ -47,6 +47,8 @@ jobs:
     if: ${{ always() }}
     name: Commit Status Final
     runs-on: ubuntu-24.04
+    permissions:
+      statuses: write
```

Comment:
Why do we need to change this workflow given that is working for all other workflows?

Reply:
For reference, the action called by this workflow requires `statuses: write` ([documented here](https://github.com/myrotvorets/set-commit-status-action/blob/master/README.md)), so while the exact default permissions may vary depending on [our repository settings](https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/#setting-the-default-permissions-for-the-organization-or-repository), I don't see any harm in making this permission explicit and reducing any excess permissions the workflow may have.